qiskit_nature/circuit/library/initial_states/_helper.py

```diff
@@ -0,0 +1,197 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021, 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Private helper functions for initial states."""
+
+from typing import Iterable, List, Sequence, Tuple, Union
+
+import numpy as np
+
+from qiskit import QuantumRegister
+from qiskit.circuit import Gate, Qubit
+from qiskit.circuit.library import RZGate, XGate, XXPlusYYGate
+
+
+def _prepare_slater_determinant_jordan_wigner(  # pylint: disable=invalid-name
+    register: QuantumRegister, transformation_matrix: np.ndarray
+) -> Iterable[Tuple[Gate, Tuple[Qubit, ...]]]:
+    """Prepare a Slater determinant under the Jordan-Wigner Transform."""
```

Comment:
I know that this is a private utility method but since you started a docstring please complete it with function argument and return type documentation.

Reply:
Done.